python/GafferSceneUITest/ShaderViewTest.py

```diff
@@ -89,17 +89,17 @@ def testRegisterReferenceScene( self ) :
 		s["b"]["assignment"]["shader"].setInput( s["b"]["shader"] )
 		s["b"]["out"].setInput( s["b"]["assignment"]["out"] )
 
-		s["b"].exportForReference( self.temporaryDirectory() + "/test.grf" )
+		s["b"].exportForReference( self.temporaryDirectory() / "test.grf" )
 
-		GafferSceneUI.ShaderView.registerScene( "test", "Default", self.temporaryDirectory() + "/test.grf" )
+		GafferSceneUI.ShaderView.registerScene( "test", "Default", ( self.temporaryDirectory() / "test.grf" ).as_posix() )
```

Comment:
Rather than use `as_posix()` here, we should change `registerScene()` to take `const std::filesystem::path &`.

Reply:
Nice to get one more `filesystem::path` in place! Changed in b4b24290490741c9d1da338089f0754b51006781